core/embed/trezorhal/stm32u5/secure_aes.c

```diff
@@ -136,8 +290,28 @@ secbool secure_aes_ecb_encrypt_hw(const uint8_t* input, size_t size,
   return sectrue;
 }
 
+secbool secure_aes_init(void) {
```

Comment:
@TychoVrahe What do you think about relocating `secure_aes_init()` either before or after the encrypt and decrypt routines? It might be a bit confusing when placed between them.

Reply:
moved to the end https://github.com/trezor/trezor-firmware/commit/30248cbd49fd9e8d0a8fc2d4ccff7e3b1622d9a9